substrate/frame/src/lib.rs

```diff
@@ -51,317 +51,11 @@
 //! Please note that this crate can only be imported as `polkadot-sdk-frame` or `frame`.
 
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg(feature = "experimental")]
 
-#[doc(no_inline)]
-pub use frame_support::pallet;
+// NOTE that we put the actual code into another module to keep the no-std capability of this create
+// even when the `experimental` feature is disabled.
```

Comment:
I don't get this comment. Also generally it doesn't make sense that the `experimental` feature disables everything of this crate. 

What is the purpose of this feature then? Just don't use the crate if you don't want to use experimental stuff.

Reply:
Okay i reverted the changes to the FRAME crate. Apparently it is not published anyway - so should not be needed for the umbrella crate until then.  
Also increased the warning in the docs. I could not rename it since the name is used in some crate-access generation macros and repeated often in docs... can fix in the future once we start publishing.